deploy/kuberhealthy.yaml

```diff
@@ -191,6 +191,16 @@ spec:
     spec:
       serviceAccountName: kuberhealthy
       automountServiceAccountToken: true
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
```

Comment:
Since this spec will be installed on unknown clusters, I'd recommend we use `preferredDuringSchedulingIgnoredDuringExecution` with a high weight instead.  Using `requiredDuringSchedulingIgnoredDuringExecution` would prevent Kuberhealthy from spinning up at all on clusters with a single node or clusters that only have a single node with schedulable space which I don't think we want as default behavior. 

You can read about weights [here](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/scheduling/nodeaffinity.md#nodeaffinity) but higher = more preference.  An example of the format is [here](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#an-example-of-a-pod-that-uses-pod-affinity).

Other than that, this looks great!  Once the `required`>`preferred` change is made I'll merge it in.

Additionally, we maintain [helm charts](https://github.com/helm/charts/tree/master/stable/kuberhealthy/templates) for Kuberhealthy, you should make a PR for these changes in all the configs there, too!

Reply:
Yeahhh `preferredDuringSchedulingIgnoredDuringExecution` makes more sense! 

Will create another PR for the changes in the helm chart! Thanks for the speedy review!